### ISSUE TITLE
Why a work in progress?

### ISSUE THREAD

**Issue by Steve-Wax** [jfedor2/twist-to-scroll-trackball#4]
I like your design.  It looks like a cross between the Logitech Trackman Marble Mouse and the Kensington Slim Blade with the best features of each.  I'm thinking of building one.  You mention that the scrolling is a work in progress.  I'm curious why you think that is and what needs to be improved.  Do you think it is good for regular use or should I plan to do some work on it first.  Thanks for the design.

**Comment by jfedor2** [comment]
Thank you for the kind words.

I think the scroll works pretty well as it is, but it should ideally be other users to confirm that as I may be a bit biased. I wrote it's a work in progress because it's a bit of a heuristic, the code has to decide whether the user intended to scroll or just make normal cursor tracking motions. So there's some wiggle room for tweaking the constants and it might even be up to personal preference.

But don't let that discourage you from building one, I think it's perfectly usable in its current form. The work-in-progress part just means I make no bold claims about some of the arbitrary decisions necessarily being the optimal ones.

**Comment by Steve-Wax** [comment]
Makes sense.  I've ordered the boards and parts.  I'll let you know what I think once they arrive and I have a chance to build it.

**Comment by Steve-Wax** [comment]
So I made two of these and have been using them for a few weeks.  Generally I think it is good.  I'm having trouble with the scroll from time to time.  For some time it wasn't recognizing when I wanted to scroll and occasionally it was scrolling when I didn't want it to.  I'm wondering if it is the finish on my ball because the two different mice were acting differently.  I will keep using and experimenting to see if I can figure out what is going on.

The rest is working well.  I did the USB-C mod that  @JonnyHaystack designed.  The only change I would make is putting a ridge on the lower buttons as I can't feel when I'm on the lower button or not.  Thanks for the design.
![IMG_1577](https://user-images.githubusercontent.com/52845023/176989270-1b4b77d1-666f-47a3-bb53-eb363dc2155f.JPG)

**Comment by itsnoteasy** [comment]
I had trouble with the cursor getting intermittent, this worked for me. although lift off detection is important in mouses, it can make trackballs harder to use. you can add these lines to pmw3360.c, near the end, under //set initial cpi resolution

    // LOD: 0x00 disable lift detection, 0x02 = 2mm, 0x03 = 3mm
    write_register(Lift_Config, 0x00);
